Musigwa/SendIT-API
# #162114302 user can log into her/his account

## What does this PR do (?)

It helps the user to log into his/her account.

## Description of Task to be completed (?)

The following endpoint should be accessible to help the user login.

- POST /auth/login

## How should this be manually tested (?)

After cloning this repo and switch to this branch, run the dependencies installation using any Node package manager,

- Using any RESTful Client tool (Usually the postman), Provide the required login information within the request body: email, and password.
- If the provided information is correct, you should get the response containing the access token within the response body.

Key: Content-Type value: application/json

## Any background context you want to provide (?)

I'm using ES6 features in this branch, please be aware of the transpiler (.babelrc file) configurations.

## What are the relevant pivotal tracker stories (?)

1. 162114302
